src/main/java/ca/corefacility/bioinformatics/irida/plugin/amrdetection/StarAMRPlugin.java

```diff
@@ -0,0 +1,60 @@
+package ca.corefacility.bioinformatics.irida.plugin.staramr;
```

Comment:
In order for the location of this file to match the package it should be moved to a directory named `staramr` instead of `amrdetection`. That is the file should be moved to `staramr/StarAMRPlugin.java` from `amrdetection/StarAMRPlugin.java`.